src/blue.ts

```diff
@@ -60,10 +60,16 @@ function createBlueShadowTarget(target: BlueProxyTarget): BlueShadowTarget {
     if (isFunction(target)) {
         // this new shadow target function is never invoked just needed to anchor the realm
         try {
-            shadowTarget = 'prototype' in target ? function () {} : () => {};
+            if ('prototype' in target) {
+                shadowTarget = hasOwnProperty(target, 'arguments')
+                    ? function () {}
```

Comment:
if this library is written as a module... how come this line will ever be different to the next one? both functions will be strict IMO.

Reply:
I believe what we want is to always produce a strict function as a shadow target, and I believe that was implicit based on the assumption that we are running in strict mode anyways.

Reply:
The issue we ran into was that classes or strict mode functions didn't align with the shadow target which used to be a sloppy function (assuming our secure environment isn't bundled as strict)

Reply:
So the inference here is to check for a sloppy function and provide a sloppy target else a strict function